starter_code/script.js

```diff
@@ -1,9 +1,105 @@
-window.onload = function() {
-  document.getElementById("start-button").onclick = function() {
-    startGame();
+// function Draw() {
+//   this.ctx = document.getElementById('canvas').getContext('2d');
+//   // draw background
+//   this.ctx.fillStyle = '#A9A9A9';
+//   this.ctx.fillRect(0, 0, 400, 800);
+// }
+
+
+// function startGame() {
+//   Draw();
+// }
+
+
+
+
+function startGameWay() {
+  var canvas = document.getElementById('road');
+  var ctx = canvas.getContext('2d');
+  ctx.fillRect(0, 0, 400, 800);
+  ctx.fillStyle = '#A9A9A9';
```

Comment:
You need to set the `fillStyle` before calling `fillRect`, the rectangle style is the one that was defined before. That's why when you move your car the road becomes red